test/repeater/repeater.js

```diff
@@ -353,6 +353,96 @@ describe('test/repeater/repeater.js >', () => {
     });
   });
 
+  describe('pause and resume >', () => {
+    it('OK even when tracker is empty', (done) => {
+      let _tracker = repeater.pauseGenerators();
+      expect(_tracker).to.deep.equal({});
+      _tracker = repeater.resumeGenerators();
+      expect(_tracker).to.deep.equal({});
+      done();
+    });
+
+    it('pause and resume should pause and resume the task function ' +
+      'passed in', (done) => {
+      let count1 = 0;
+      let count2 = 0;
+      function stub1() {
+        count1++;
+      }
+
+      function stub2() {
+        count2++;
+      }
+
+      repeater.create({
+        name: 'Generator_Pause_Resume_1',
+        interval: 1,
+        func: stub1,
+      });
+
+      repeater.create({
+        name: 'Generator_Pause_Resume_2',
+        interval: 1,
+        func: stub2,
+      });
+
+      let count1BeforePause;
+      let count2BeforePause;
+      setTimeout(() => {
+        // proves start was called and the task was executed
+        expect(count1).to.be.at.least(1);
+        expect(count2).to.be.at.least(1);
+
+        count1BeforePause = count1;
+        count2BeforePause = count2;
+
+        // pause the repeat
+        repeater.pauseGenerators();
```

Comment:
I could see this causing test flapping. Maybe increase the interval to be safe?

Reply:
I was thinking the repeat interval on line 385.

Reply:
All the repeats are set to run immediately after are they are created. The interval at L385/L379 defines how long the repeat has to pause before starting the next one. Increasing it might cause flappers for the asset in L415.

Reply:
The interval is in ms, right? So the repeater is running once every ms, and count1 will be 30 by the time it's paused. That seems excessive. I don't see how that could cause flapping for any of the later "expect"s because the margin of error is so big. I do think it could potentially cause flapping for the L405 "expect" if the repeater runs right after count1BeforePause is set.

Reply:
Yes!! I got what you are trying to say. Will increase the interval.